src/Microsoft.AspNetCore.Routing/DefaultInlineConstraintResolver.cs

```diff
@@ -18,6 +18,7 @@ namespace Microsoft.AspNetCore.Routing
     public class DefaultInlineConstraintResolver : IInlineConstraintResolver
     {
         private readonly IDictionary<string, Type> _inlineConstraintMap;
+        private readonly IServiceProvider _serviceProvider;
```

Comment:
Ignore changes in `DefaultInlineConstraintResolver`. My plan is for this resolve to reuse the `ParameterPolicyActivator` internally